src/test/groovy/com/github/dispader/manifesto/VersionSpec.groovy

```diff
@@ -4,12 +4,34 @@ import spock.lang.*
 
 class VersionSpec extends Specification {
 
-    def 'can determine if project is controlled by Git'() {
+    def 'handles no Git repository'() {
         when:
-            Version.metaClass.static.getGit = { null }
+            Version.metaClass.static.getHasRepo = { false }
         then:
-            Version.version == '0.0.0'
+            Version.warningText == Version.MSG_NO_REPO
```

Comment:
If we did actually change the name of `Version` to `Repository`, this might become `Repository.MSG_NOT_GIT`, which reads kinda nice.